.github/scripts/build_info.py

```diff
@@ -151,39 +153,43 @@ def get_changed_python_dependencies_files() -> List[str]:
     return changed_dependencies_files
 
 
-def should_test_all_python_versions() -> bool:
+def check_if_pr_has_label(action: str, label: str):
```

Comment:
nit: I think it's a bit cleaner to have the `label` being checked as the first argument as `action` is only used to improve human-readability